wemake_python_styleguide/visitors/ast/operators.py

```diff
@@ -225,6 +225,22 @@ def visit_NamedExpr(
         self,
         node: ast.NamedExpr,
     ) -> None:
-        """Disallows walrus ``:=`` operator."""
-        self.add_violation(consistency.WalrusViolation(node))
+        """Disallows walrus ``:=`` operator outside comprehension."""
+        closest = walk.get_closest_parent(
+            node,
+            (
+                ast.ListComp,
```

Comment:
Please, move this to be a class level constant.

Reply:
Do you mean a whole tuple with possible comprehensions?

Reply:
Yes, like we do with other `ClassVar[AnyNodes]`

Reply:
Pushed :)